openedx_events/learning/data.py

```diff
@@ -425,3 +425,27 @@ class DiscussionThreadData:
     user_course_roles = attr.ib(type=List[str], factory=list)
     user_forums_roles = attr.ib(type=List[str], factory=list)
     options = attr.ib(type=dict, factory=dict)
+
+
+@attr.s(frozen=True)
+class CourseNotificationData:
+    """
+    Attributes defined for Open edX Course Notification data object.
+
+    Arguments:
+        course_key (str): identifier of the Course object.
+        app_name (str): name of the app requesting the course notification.
+        notification_type (str): type of the notification.
+        content_url (str): url of the content the notification will redirect to.
+        content_context (dict): additional information related to the content of the notification.
+        audience_filters (dict): additional information related to the audience of the notification.
```

Comment:
Please, explain this a little more so we have more context. You can add examples for both arguments perhaps

Reply:
I have added examples for both the arguments!